src/Core/Properties/AssemblyInfo.cs

```diff
@@ -4,4 +4,4 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("Tests.IQsharp" + SigningConstants.PUBLIC_KEY)]
-[assembly: InternalsVisibleTo("Microsoft.Quantum.IQSharp.Jupyter" + SigningConstants.PUBLIC_KEY)]
\ No newline at end of file
+[assembly: InternalsVisibleTo("Microsoft.Quantum.IQSharp.Kernel" + SigningConstants.PUBLIC_KEY)]
```

Comment:
This name change makes sense, but will affect signing process such that we'll have to adapt slightly. Please feel free to sync with me offline if I can help with that adaptation; I'd suggest an end-to-end build at the least to make sure that this renaming doesn't break anything across QDK repos.

Reply:
End-to-end build completed successfully with latest Jupyter Core release.